pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -220,6 +221,16 @@ func (c *Cluster) Start() error {
 }
 
 func (c *Cluster) startProvisioningOverPVCs(config *provisionConfig) {
+	//check k8s version
+	k8sVersion, err := k8sutil.GetK8SVersion(c.context.Clientset)
+	if err != nil {
+		logger.Warningf("error finding Kubernetes version. %+v", err)
```

Comment:
this should be an error if we're aborting the provisioning

Reply:
added this as a error in the config.